### DISCUSSION TITLE
HELP "Unknown error building nav mesh"

### DISCUSSION THREAD

**Discussion by vrgallery02** [mozilla/hubs#5077] [Spoke Questions]
I keep getting this error message when I try to export my room in Spoke

I am using a template made by the spoke team for the room and have inserted 10 .glb 3D models of a mannequin wearing clothes my students have developed in the program Clo. I originally had 20 of these .glb files but reduced to 10 and I am having the same error, so I am thinking it might be an issue with the room not the file size? 

![image](https://user-images.githubusercontent.com/98921261/152244918-2f68e203-1f12-4ec2-b615-ac21f989af91.png)

**Comment by cvanmeurs** [comment]
Which template are you using?  I typically only see navmesh errors when someone tries to use a model that is way too large and out of scale (such as a terrain that is several kilometers wide).  Do you still get the error if you take ALL of the imported student models out?

**Reply by maxime-memtell** [reply]
That was the correct error in my case. I had used the "Crater" model but scaled it x4.

It's worth noting that deleting or re-scaling it wasn't enough, I also had to reload the page/project for it to work.

Perhaps there's an opportunity to improve error messages if this error is often due to that.
